### ISSUE TITLE
Incorporate something like npm-run-all that allows for running multiple scripts synchronous  or in parallel (ie `generate:*`)

### ISSUE THREAD

**Issue by chrisbull** [antfu-collective/ni#243]
### Clear and concise description of the problem

npm, yarn, or pnpm  all don't have a easy way to run multiple scripts synchronous or in parallel. 
For example, if I want to run a single generate script, but there are 3 different generate scripts. I would need to do the following:

```json
{
    "scripts": {
        "generate": "npm run generate:one && npm run generate:two && npm run generate:three",
        "generate:one": "npm run do-a-thing 1",
        "generate:two": "npm run do-a-thing 2",
        "generate:three": "npm run do-a-thing 3",
    }
}
   
```

Obviously this becomes a pain.

So `npm-run-all` has a solution `run-s generate:*`

```json
{
    "scripts": {
        "generate": "run-p generate:*",
        "generate:one": "npm run do-a-thing 1",
        "generate:two": "npm run do-a-thing 2",
        "generate:three": "npm run do-a-thing 3",
    }
}
```

But this means I need to also install npm-run-all 😞 

### Suggested solution

Solution would be to have this package already include and embeded into `nr`

I'm thinking something like `nr-s` and `nr-p`

So I would from terminal or by script run `nr-p generate:*` or `nr generate` which execs `nr-p generate:*` from package.json

```json
{
    "scripts": {
        "generate": "nr-p generate:*",
        "generate:one": "nr do-a-thing 1",
        "generate:two": "nr do-a-thing 2",
        "generate:three": "nr do-a-thing 3",
    }
}
```

This would be amazing! Thanks in advance!

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by antfu** [comment]
This is a technical challenge to implement correctly and robustly - I would suggest you to use `npm-run-all` which is popular and handles many edge cases for you. ni is more like an alias, and such a feature would be considered out-of-scope.

Thanks for bringing this up
